masterlist.yaml

```
@@ -8057,6 +8057,45 @@ plugins:
       - crc: 0xA082C1A4
         util: 'TES4Edit v4.0.3'
 
+  - name: 'Rathunas(01|-MidasMagic|-BetterMusicSystem)\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/45880/'
+        name: 'Rathunas'
+  - name: 'Rathunas01.esp'
+    req:
+      - 'Cobl Main.esm'
+      - 'DLCShiveringIsles.esp'
+    msg:
+      - type: say
+        content:
+          - lang: en
+            text: 'Requires: Unique Landscapes-Brena River Ravine'
+          - lang: de
+            text: 'Benötigt: Unique Landscapes-Brena River Ravine'
+      - *requiresOBSEv21
```

You can remove that message, as both main files have UL as a master file.
```suggestion
    msg: [ *requiresOBSEv21 ]
```